### ISSUE TITLE
Scrambled ellipse plot

### ISSUE THREAD

**Issue by JAvRZ** [andi-nl/ANDI-frontend#181]
This datafile makes a few things go spectacularly wrong (well, the plots look very off, it does not crash or anything). (I couldnt attach a csv file, but renaming this as csv lets me read it in fine).
[Reordering.txt](https://github.com/andi-nl/ANDI-frontend/files/603666/Reordering.txt)

The values don't look particularly strange to me, but the plots do. The line plot has one extreme value, but that is all, there doesnt seem to be anything fundamentally wrong with it.
![image](https://cloud.githubusercontent.com/assets/13958834/20484516/cf0b4c80-aff7-11e6-8663-9582b9b2689f.png)

The ellipse plot though:
![image](https://cloud.githubusercontent.com/assets/13958834/20484549/f385f11e-aff7-11e6-85b2-633b6d46afbc.png)

**Comment by JAvRZ** [comment]
The flipping of axis also happens with less extreme examples, see for example
![image](https://cloud.githubusercontent.com/assets/13958834/20484605/36b18976-aff8-11e6-951f-d35cdf8591b2.png)

**Comment by JAvRZ** [comment]
EDIT: Ah sorry, I wasn't aware that the fixing commit wasn't merged yet. Ignore this comment then.

Alas, this still looks scrambled:
![image](https://cloud.githubusercontent.com/assets/13958834/21307196/b8870ac4-c5d4-11e6-88a7-5e2a621744e4.png)
Used data:
[patientdata.txt](https://github.com/andi-nl/ANDI-frontend/files/660592/patientdata.txt)
(which I downloaded from the last screen, which was very handy).
On Chrome.

(I now also dropped cache, same result)
